### DISCUSSION TITLE
Inclusão de um podcast no aplicativo

### DISCUSSION THREAD

**Discussion by Neoprot** [cerejeiros/mobile#26] [Polls]
### Podcast
Achei esse [podcast](https://open.spotify.com/show/2GFx3Gf0R4KoCaPand0hYQ) de inteligência emocional e autodesenvolvimento de manhã e pensei em incluir ele no aplicativo, seja via link, ou fazendo um player do spotify no aplicativo, se concordarmos com a ideia, posso a partir da decisão tentar entrar em contato com a pessoa que faz esse podcast para pedir permissão de incluir o podcast dela no nosso aplicativo (Espero eu que gratuito)
@cerejeiros/back-end @cerejeiros/front-end

**Comment by BeyondMagic** [comment]
Se é público no Spotify, não precisa pedir permissão. Spotify inclusive tira essa restrição para poder ter mais divulgação.

Isso a parte peça caso a ideia seja aceita porque às vezes a pessoa fica inclinada à fazer algo só pra ajudar o aplicativo.

**Comment by Neoprot** [comment]
@BeyondMagic gostaria de pedir, pq nunca se sabe se a pessoa não gostaria de só colocar sem conversar com ela

**Comment by Luizaxx** [comment]
Escutei o podcast e achei conveniente colocar no app

**Comment by limonadaquente** [comment]
Top !!

**Comment by kaioenzo** [comment]
Acredito que deve estar no nosso backlog e pode ser refinado, temos um tempo curto e talvez não caiba na primeira release, mas com certeza, é uma boa ideia!

**Comment by Neoprot** [comment]
Vou criar agora no backlog e vou tentar contato ainda essa semana com a criadora
